samples/testing.fs

```diff
@@ -0,0 +1,45 @@
+module samples.unittest
+
+open Microsoft.Azure.WebJobs
+open DurableFunctions.FSharp
+open System
+
+[<FunctionName("SayHelloTest")>]
+let SayHello([<ActivityTrigger>] name) = 
+    sprintf "Hello %s!" name
+
+[<FunctionName("HelloSequenceTest")>]
+let RunWorkflow ([<OrchestrationTrigger>] context: DurableOrchestrationContextBase) = 
+    context |>    
+    orchestrator {
+      let! hello1 = Activity.callByName<string> "SayHello" "Tokyo"
+      let! hello2 = Activity.callByName<string> "SayHello" "Seattle"
+      let! hello3 = Activity.callByName<string> "SayHello" "London"
+
+      // returns ["Hello Tokyo!", "Hello Seattle!", "Hello London!"]
+      return [hello1; hello2; hello3]
+    }
```

Comment:
Consider using a strongly typed implementation style. I don't want to give the impression that only `callByName` style is testable. (I hope others are, too)